test/js/specs/happy.js

```diff
@@ -12,7 +12,7 @@ describe('Happy Paths', options, ({driver, pageObjects}) => {
   const {documentSelection, welcome, documentUpload} = pageObjects
```

Comment:
As mention above, you could just import the locale in this file once
`const locale = (lang="en") => require(`../../../src/locales/${lang}.json`)`
Then you can do 
```
const copy = locale("es")
const welcomeLocale = copy["welcome"]
```

Reply:
Thank you for the suggestion! 🙇